command/agent/sink/file/sink_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/hashicorp/vault/command/agent/auth"
```

Comment:
Please group this import with the others below.

Reply:
Woops, thanks - auto-formatting always gets me.